tcases-lib/src/main/java/org/cornutum/tcases/util/FilenameUtils.java

```diff
@@ -0,0 +1,19 @@
+package org.cornutum.tcases.util;
+
+public class FilenameUtils
+  {
+
+  public static String getBaseName( String filepath)
```

Comment:
The code is heavily inspired by commons.io IOUtils. I added a testcase too.

Reply:
Why is this better than using the original commons-io dependency?

Reply:
So my goal was to remove the common-io compile dependency to use tcases as a library with minimal transitive dependencies (and thus minimal potential for conflicts with other libraries). commons.io seemed to be used so little that removing it seemed feasible at a low price. But you can disagree saying that it is a benign dependency and move it back in, since the method and classnames used are the same, it's just a matter of deleting 3 Java files and reverting the imports in 3 places or so.

Reply:
else this getBaseName and the commons.io.FileUtils.getBaseName methods should be equivalent.

Reply:
Agreed, the need for common-io is negligible, so I'm OK with replacing it.